src/test/migrations/steps/20240909101347-step-exclude-public-migrations.ts

```diff
@@ -27,7 +27,7 @@ export default async () => {
     table: { table: 'public', schema: 'migrations' },
   })
 
-  const query = `${pgp.helpers.insert(values, cs)}`
+  const query = `${pgp.helpers.insert(values, cs)} on conflict do nothing;`
```

Comment:
In case we double run migrations, I like to be safe
In my case, I ran public migrations and then steps, which caused `key already exists` error.

We can omit if you prefer